src/pages/App.tsx

```diff
@@ -29,6 +29,7 @@ import { RedirectDuplicateTokenIdsV2 } from './AddLiquidityV2/redirects'
 import { PositionPage } from './Pool/PositionPage'
 import AddLiquidity from './AddLiquidity'
 import ApeModeQueryParamReader from 'hooks/useApeModeQueryParamReader'
+import CreateProposal from './CreateProposal'
```

Comment:
Consider using `const CreateProposal = React.Lazy(() => import './CreateProposal')` instead to avoid increasing the size of the bundle for all users :D